assets/cubyz/biomes/ocean/beach.zig.zon

```diff
@@ -0,0 +1,27 @@
+.{
+	.properties = .{
+		.ocean,
+		.wet,
+	},
+	.minHeight = 2,
+	.maxHeight = 12,
+	.chance = 0,
+	.roughness = 1,
+	.interpolation = .linear,
+	.hills = 2,
+	.music = "cubyz:tides",
+		.structures = .{
+		.{
+			.id = "cubyz:ground_patch",
+			.block = "cubyz:sand",
+			.chance = 1,
+			.width = 4,
+			.variation = 8,
+			.depth = 4,
+			.smoothness = 0.8,
+		},
```

Comment:
Once again, could you avoid the ground patch spam in favor of making a better biome transition in the first place?

Using structures for this is really not cheap.